scripts/install_cvmfs_eessi.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Gather information about the OS
+. /etc/os-release
```

Comment:
Kinda sad that lsb went away, but /etc/os-release is probably as close as we get.